lib/rack/tracker/metrigo/metrigo.rb

```diff
@@ -0,0 +1,35 @@
+class Rack::Tracker::Metrigo < Rack::Tracker::Handler
+  class Event < OpenStruct
+    def initialize(args)
+      args[:arguments] ||= {}
+      super
+    end
+
+    def write
+      "DELIVERY.DataLogger.#{function_name}(#{write_arguments})"
+    end
+
+    def write_arguments
+      arguments.to_json  if arguments.present?
```

Comment:
yeah.  i think this method can be removed.  Before i head the case where `arguments.to_json` resulted in `null`, which was not nice.  thanks.